logistics/scripts/modules/settings/settings-management.psm1

```diff
@@ -324,9 +324,29 @@ function New-DevelopmentAppSettings([hashtable] $Settings = @{ }) {
 
         $credentialSettings = Merge-Hashtables @{ }, (Get-CredentialSettingsByProject)[$project]
         $newUserSettingsPath = Join-Path $projectPath "appsettings.user.json"
+        
         New-JsonFile $newUserSettingsPath $credentialSettings
-        $newSettingsFiles += $newUserSettingsPath
+
+         if ($projectPath -like "*EdFi.Ods.SandboxAdmin")
+         {             
+            $userSecrets += (type $newUserSettingsPath | dotnet user-secrets set --project $projectPath --id 'f1506d66-289c-44cb-a2e2-80411cc690ea')
+         }
+         elseif ($projectPath -like "*EdFi.Ods.SwaggerUI")
+         {              
+            $userSecrets +=(type $newUserSettingsPath | dotnet user-secrets set --project $projectPath --id 'f1506d66-289c-44cb-a2e2-80411cc690eb')
+         }
+         elseif ($projectPath -like "*EdFi.Ods.WebApi.NetCore")
+         {              
+            $userSecrets += (type $newUserSettingsPath | dotnet user-secrets set --project $projectPath --id 'f1506d66-289c-44cb-a2e2-80411cc690ec')
+         }
+         elseif ($projectPath -like "*EdFi.Ods.Api.IntegrationTestHarness")
+         {              
+            $userSecrets += (type $newUserSettingsPath | dotnet user-secrets set --project $projectPath --id 'f1506d66-289c-44cb-a2e2-80411cc690ed')
+         }
+
+        $newSettingsFiles += $newUserSettingsPath        
```

Comment:
does it make sense to create a hashtable with keyed on the application and guid for the secrets file?

Reply:
it's easy to add to secret json using json file which is one line change and already working fine .

Reply:
Yes  I created new function to return guid id based on project now and also directly converting hashtable to json string for loading values